htdocs/core/lib/company.lib.php

```diff
@@ -1830,3 +1842,73 @@ function show_subsidiaries($conf, $langs, $db, $object)
 
 	return $i;
 }
+/**
+ * 		Add Event Type SQL
+ *
+ *		@param	string		$sql		    $sql modified
+ * 		@param	string	    $actioncode		Action code
+ * 		@param	string		$donetodo		donetodo
+ * 		@param	string		$now		    now
+ * 		@param	string		$filters		array
+ * 		@param	string		$donetodo		donetodo
+ * 		@param	string		$sqlANDOR		"AND", "OR" or ""
+ * 		@return	void
```

Comment:
For code consistency, we try to avoid function that return "void".
Can you modify this method to return instead $sql (no need to send it as an input parameter, returning the string to concat like done into addMailingEventTypeSQL should be enought)